index.html

```diff
@@ -56,6 +56,17 @@
           type: "bar",
           data: data,
           options: {
+            onClick: (_, [raw]) => {
+              try {
+                const commit = raw.element.$context.raw.x;
+                window.open(
+                  `https://github.com/MetaMask/metamask-extension/commit/${commit}`,
+                  "_blank"
+                );
+              } catch (error) {
+                console.error(error);
```

Comment:
Curious what type of error we might expect here.  If we're worried this property might not exist, we could do:

```
const commit = raw?.element?.$context?.raw?.x
```

Reply:
Yes I made the check on commit